examples/long-term-schedule-transaction.js

```diff
@@ -0,0 +1,198 @@
+// LongTermScheduledTransactionExample.js
+
+import {
+    Client,
+    AccountCreateTransaction,
+    Hbar,
+    PrivateKey,
+    KeyList,
+    TransferTransaction,
+    ScheduleSignTransaction,
+    ScheduleInfoQuery,
+    AccountBalanceQuery,
+    AccountUpdateTransaction,
+    Timestamp,
+} from "@hashgraph/sdk";
+
+import dotenv from "dotenv";
+dotenv.config();
+
+async function main() {
+    console.log("Long Term Scheduled Transaction Example Start!");
+
+    // Step 0: Create and configure the SDK Client.
+    const operatorId = process.env.OPERATOR_ID;
+    const operatorKey = PrivateKey.fromStringED25519(process.env.OPERATOR_KEY);
+    const client = Client.forName(process.env.HEDERA_NETWORK || "testnet");
+    client.setOperator(operatorId, operatorKey);
+
+    // Step 1: Create key pairs
+    const privateKey1 = PrivateKey.generateED25519();
+    const privateKey2 = PrivateKey.generateED25519();
+    const thresholdKey = new KeyList([
+        privateKey1.publicKey,
+        privateKey2.publicKey,
+    ]);
+    console.log("Created a Key List: ", thresholdKey);
+
+    // Step 2: Create the account
+    console.log("Creating new account...");
+    const alice = (
```

Comment:
Rename `const alice` to `const aliceId` to make it clear that this represents the ID.